src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue

```diff
@@ -3,20 +3,17 @@ tm-li-transaction(:color="color" :time="transaction.time" :block="transaction.he
   template(v-if="delegation")
     div(slot="caption")
       | Delegated&nbsp;
-      b {{tx.delegation.amount}}
-      span &nbsp;{{tx.delegation.denom.toUpperCase()}}S
+      b {{ prettify(tx.delegation.amount) }}
+      span &nbsp;{{ bondingDenom }}s
```

Comment:
there should be an apostrophe if there is an added `s` 

...

`'s`

Reply:
yes, the bonding denom is in singular. Although we should check the case when you're delegating only `1` and remove the `s`

Reply:
edited